selftests/unit/test_job.py

```diff
@@ -130,6 +130,25 @@ def post_tests(self):
         self.assertEqual(myjob.unique_id[::-1],
                          open(os.path.join(myjob.logdir, "reversed_id")).read())
 
+    @unittest.skip("Issue described at https://trello.com/c/qgSTIK0Y")
```

Comment:
Well is it a problem? The tests are passing and to avoid files created by unittest you can make Job using `tempfile` dir for log_dir and remove it afterwards (and if you insist on unittesting empty `Job`, you can mock it, but I don't think it's necessary...)